UnityProject/Assets/Scripts/Items/Implants/Organs/SlimeCore.cs

```diff
@@ -0,0 +1,222 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Chemistry;
+using Chemistry.Components;
+using HealthV2;
+using HealthV2.Living.PolymorphicSystems.Bodypart;
+using Systems.Character;
+using UI.CharacterCreator;
+using UnityEngine;
+
+[RequireComponent( typeof(ReagentCirculatedComponent))]
+public class SlimeCore : BodyPartFunctionality
```

Comment:
namespace your files. If you don't know how to, let your IDE handle it for you.

Reply:
I can't seem to find the option in rider, used to be just able to right click on the class and it would say move to namespace, but it's no longer visible